desktop-src/directcomp/initialize-directcomposition.md

```diff
@@ -544,7 +544,7 @@ HRESULT DemoApp::Initialize()
     wcex.cbClsExtra    = 0;
     wcex.cbWndExtra    = sizeof(LONG_PTR);
     wcex.hInstance     = HINST_THISCOMPONENT;
-    wcex.hbrBackground = (HBRUSH)(COLOR_WINDOW+1);;
+    wcex.hbrBackground = (HBRUSH)(COLOR_WINDOW+1);
     wcex.lpszMenuName  = nullptr;
     wcex.hCursor       = LoadCursor(NULL, IDC_ARROW);
     wcex.lpszClassName = L&quot;DirectCompDemoApp&quot;;
```

Comment:
I would just throw in a quick last question. Since the line above contains an error that would make the code snippet fail to compile (the `&quot;`), I would assume it's more efficient to do the fix en masse internally.

Hence, am I right that an external PR to fix such issues (regardless of whether said PR fixes all occurrences) is not recommended?

Reply:
I feel like it'd be more efficient to fix those HTML entity problems internally.